nasm/README.md

```diff
@@ -0,0 +1,25 @@
+# Netwide Assembler
+
+This is the netwide assembler and it is meant to be used in the binary compatibility mode of unikraft.
+
+## Content
+- sample_code.s: a sample assembly code file that can be assembled using nasm.
+- usr/bin/nasm: the nasm binary
+- lib/ & lib64/: the required libraries for nasm
+
+## Extract dynamic libraries
+```../extract.sh usr/bin/nasm```
```

Comment:
Use triple backtics on a separate line. Use `console` after triple backticks for commands.